gatsby-node.js

```diff
@@ -232,22 +287,6 @@ exports.onPostBuild = async function (
   activity.end();
 };
 
-/**
- * Copy the settings, synonyms, and rules of the source index to the target index
- * @param client
- * @param sourceIndex
- * @param targetIndex
- * @return {Promise}
- */
-async function scopedCopyIndex(client, sourceIndex, targetIndex) {
```

Comment:
I removed this since it wasn't being used but I can restore it if it should be included

Reply:
yep, I noticed that but forgot to commit another time as well, thanks!